public/app/plugins/panel/candlestick/CandlestickPanel.tsx

```diff
@@ -255,73 +257,100 @@ export const CandlestickPanel = ({
-              <AnnotationEditorPlugin data={alignedDataFrame} timeZone={timeZone} config={config}>
-                {({ startAnnotating }) => {
+            {config.featureToggles.newVizTooltips ? (
+              <TooltipPlugin2
+                config={uplotConfig}
+                hoverMode={TooltipHoverMode.xAll}
+                queryZoom={onChangeTimeRange}
+                clientZoom={true}
+                render={(u, dataIdxs, seriesIdx, isPinned = false) => {
                   return (
-                    <ContextMenuPlugin
-                      data={alignedDataFrame}
-                      config={config}
-                      timeZone={timeZone}
-                      replaceVariables={replaceVariables}
-                      defaultItems={
-                        enableAnnotationCreation
-                          ? [
-                              {
-                                items: [
-                                  {
-                                    label: 'Add annotation',
-                                    ariaLabel: 'Add annotation',
-                                    icon: 'comment-alt',
-                                    onClick: (e, p) => {
-                                      if (!p) {
-                                        return;
-                                      }
-                                      startAnnotating({ coords: p.coords });
-                                    },
-                                  },
-                                ],
-                              },
-                            ]
-                          : []
-                      }
+                    <TimeSeriesTooltip
+                      frames={[info.frame]}
+                      seriesFrame={alignedDataFrame}
+                      dataIdxs={dataIdxs}
+                      seriesIdx={seriesIdx}
+                      mode={TooltipDisplayMode.Multi}
+                      isPinned={isPinned}
                     />
                   );
                 }}
-              </AnnotationEditorPlugin>
-            ) : (
-              <ContextMenuPlugin
-                data={alignedDataFrame}
-                config={config}
-                timeZone={timeZone}
-                replaceVariables={replaceVariables}
-                defaultItems={[]}
               />
+            ) : (
+              <>
+                <ZoomPlugin config={uplotConfig} onZoom={onChangeTimeRange} withZoomY={true} />
+                <TooltipPlugin
+                  data={alignedDataFrame}
+                  config={uplotConfig}
+                  mode={TooltipDisplayMode.Multi}
+                  sync={sync}
+                  timeZone={timeZone}
+                />
+              </>
+            )}
+            {/* Renders annotation markers*/}
+            {data.annotations && (
+              <AnnotationsPlugin annotations={data.annotations} config={uplotConfig} timeZone={timeZone} />
```

Comment:
Yes - that was the reasoning, it's under a FF, and annotations are currently wip, so that shouldn't impact any of the current functionality and we'll allow us to have them merged and then plug in the annotations.